action/steps/build.sh

```diff
@@ -123,6 +123,13 @@ else
   echo "Build failed, with exit code $BUILD_EXIT_CODE";
 fi
 
+# Add permissions to make app runnable
+if [[ "$BUILD_TARGET" == "StandaloneOSX" ]]; then
+  ADD_PERMISSIONS_PATH=$BUILD_PATH_FULL/StandaloneOSX.app/Contents/MacOS/*
```

Comment:
Do all contents of that folder have to be given the executable bit?

Reply:
So far, I've only seen a single file within that directory, i.e.
```
$BUILD_PATH_FULL/StandaloneOSX.app/Contents/MacOS/<my project name>
```

I used the chmod command with `*` based on recommended command in the issue. @dyarosla what was your experience? Thanks!

Reply:
Ok fair enough.

Sorry to not bring this up in the initial review but `StandaloneOSX` comes from targetPlatform if i'm not mistaken and .app is added by unity itself?